recipes/gneiss/meta.yaml

```diff
@@ -0,0 +1,33 @@
+package:
+  name: gneiss
+  version: 0.1.0
+
+source:
+  git_url: https://github.com/biocore/gneiss.git
+
+
+requirements:
+  build:
+    - python
+    - scikit-bio ==0.4.2
```

Comment:
Remember, 0.5.0 is broken with the shear bug.

We'll update this once scikit-bio 0.5.1 gets out.

On Tue, Aug 23, 2016 at 11:02 AM, Daniel McDonald <notifications@github.com>
wrote:

> In recipes/gneiss/meta.yaml
> <https://github.com/biocore/conda-recipes/pull/65#discussion_r75919418>:
>
> > @@ -0,0 +1,33 @@
> > +package:
> > +  name: gneiss
> > +  version: 0.1.0
> > +
> > +source:
> > +  git_url: https://github.com/biocore/gneiss.git
> > +
> > +
> > +requirements:
> > +  build:
> > +    - python
> > +    - scikit-bio ==0.4.2
>
> 0.4.2 and not 0.5.0?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/biocore/conda-recipes/pull/65/files/5547d9c5515ec842995a4be6a280122c499df246#r75919418>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AD_a3cn3bn0X1scCelT1S2ERRzOnUEBFks5qizW2gaJpZM4JrJlQ>
> .
>